js/apps/system/_admin/aardvark/APP/frontend/js/views/queryManagementView.js

```diff
@@ -233,6 +233,28 @@
               '',
               ''
             ]);
+          } else {
+            // sort by query id, descending
+            rowsArray = rowsArray.sort(function(l, r) {
+              // normalize both inputs to strings (in case they are numbers)
+              l = l[0];
+              if (typeof l !== "string") {
+                l = String(l);
+              }
+              r = r[0];
+              if (typeof r !== "string") {
+                r = String(r);
+              }
+              
+              // pad strings to the same length, by prepending '0' chars
+              l = Array(20 - l.length).join("0") + l;
+              r = Array(20 - r.length).join("0") + r;
```

Comment:
Can be simplified to:
```suggestion
              l = String(l[0]).padStart(19, "0");
              r = String(r[0]).padStart(19, "0");
```
Not sure if the total length of 19 characters was intended.

Reply:
Thanks, @Simran-B! Very helpful!